app/views/main_layout.scala.html

```diff
@@ -58,6 +59,16 @@
     ))
 }
 
+@content = {
```

Comment:
Use an implementation similar to https://github.com/hmrc/coronavirus-jrs-calculator-frontend/blob/master/app/views/templates/GovukLayoutWrapper.scala.html#L78-L86